package.json

```diff
@@ -190,6 +190,7 @@
     "ng-mocks": "^10.5.4",
     "ng-packagr": "^13.2.1",
     "node-sass": "^4.14.1",
+    "nx": "13.10.5",
```

Comment:
why 13 an not "nx": "14.4.2"

Reply:
as we discussed we will use 13 version with angular 13